WBooks/src/app/screens/Wishlist/composition.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
I think this file is pretty similar to the one used to compose the `BookList`, why don't you extract it?